infinispan-embedded/src/main/java/org/hibernate/ogm/datastore/infinispan/logging/impl/Log.java

```diff
@@ -51,6 +51,10 @@
 			+ " Enable this property explicitly in your configuration to resolve this warning.")
 	void clusteringHashGroupsMustBeEnabled(String cacheName);
 
-	@Message(id = 1108, value = "Sequences or di generation is not supported for local caches")
+	@Message(id = 1108, value = "Sequences or id generation is not supported for local caches")
 	HibernateException counterCannotBeCreatedForLocalCaches();
+
+	@Message(id = 1109, value = "Sequences or id generation is not supported if global configuration is not enabled")
```

Comment:
Maybe we can add more details to the message:

Something like,  "Counter <counterName> is not defined and cannot be created. Global persistent-location is missing in the Infinispan configuration"